### PR TITLE
cp_380 & cp_388 recoveries

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
cp_388
cp_380

File: deployment/CP05MOAS-GL380_Deploy.csv [hunk 1]
```diff
@@ -59,9 +59,9 @@ TS20201121,,TS20210131,,CP05MOAS-GL380-02-FLORTM000,10,1,2020-11-21T17:44:00,202
...
\ No newline at end of file
+TS20211214,,TS20220221,,CP05MOAS-GL380-00-ENG000000,11,1,2021-12-15T07:36:00,2022-02-22T08:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGCON-GLDRCP-00380,39.99217,-70.30902,,1000,1000,FZ-1,,
+TS20211214,,TS20220221,,CP05MOAS-GL380-01-ADCPAM000,11,1,2021-12-15T07:36:00,2022-02-22T08:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-50867,39.99217,-70.30902,,1000,1000,FZ-1,,
+TS20211214,,TS20220221,,CP05MOAS-GL380-02-FLORTM000,11,1,2021-12-15T07:36:00,2022-02-22T08:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-FLORTM-03185,39.99217,-70.30902,,1000,1000,FZ-1,,
+TS20211214,,TS20220221,,CP05MOAS-GL380-03-CTDGVM000,11,1,2021-12-15T07:36:00,2022-02-22T08:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-00073,39.99217,-70.30902,,1000,1000,FZ-1,,
+TS20211214,,TS20220221,,CP05MOAS-GL380-04-DOSTAM000,11,1,2021-12-15T07:36:00,2022-02-22T08:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00496,39.99217,-70.30902,,1000,1000,FZ-1; top mount,,
+TS20211214,,TS20220221,,CP05MOAS-GL380-05-PARADM000,11,1,2021-12-15T07:36:00,2022-02-22T08:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50244,39.99217,-70.30902,,1000,1000,FZ-1,,
```
Comment (snwhite) [inline deployment/CP05MOAS-GL380_Deploy.csv:67]: Looks good.

File: deployment/CP05MOAS-GL388_Deploy.csv [hunk 2]
```diff
@@ -65,9 +65,9 @@ TS20210517,,TS20210704,,CP05MOAS-GL388-02-FLORTM000,11,1,2021-05-17T20:25:00,202
...
\ No newline at end of file
+TS20220104,,TS20220221,,CP05MOAS-GL388-00-ENG000000,12,1,2022-01-04T20:57:00,2022-02-22T09:16:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.94245,-70.38123,,1000,1000,FZ-2,,
+TS20220104,,TS20220221,,CP05MOAS-GL388-01-ADCPAM000,12,1,2022-01-04T20:57:00,2022-02-22T09:16:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-43106,39.94245,-70.38123,,1000,1000,FZ-2,,
+TS20220104,,TS20220221,,CP05MOAS-GL388-02-FLORTM000,12,1,2022-01-04T20:57:00,2022-02-22T09:16:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-04471,39.94245,-70.38123,,1000,1000,FZ-2,,
+TS20220104,,TS20220221,,CP05MOAS-GL388-03-CTDGVM000,12,1,2022-01-04T20:57:00,2022-02-22T09:16:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-09059,39.94245,-70.38123,,1000,1000,FZ-2,,
+TS20220104,,TS20220221,,CP05MOAS-GL388-04-DOSTAM000,12,1,2022-01-04T20:57:00,2022-02-22T09:16:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00341,39.94245,-70.38123,,1000,1000,FZ-2; top mount,,
+TS20220104,,TS20220221,,CP05MOAS-GL388-05-PARADM000,12,1,2022-01-04T20:57:00,2022-02-22T09:16:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50151,39.94245,-70.38123,,1000,1000,FZ-2,,
```
Comment (snwhite) [inline deployment/CP05MOAS-GL388_Deploy.csv:73]: Looks good.

**Review by snwhite — APPROVED** [review]

File: deployment/CP05MOAS-GL388_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL388_Deploy.csv:73]: good

File: deployment/CP05MOAS-GL380_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL380_Deploy.csv:67]: good

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**